templates/cassiopeia/index.php

```diff
@@ -53,6 +53,9 @@
 // Load specific language related CSS
 HTMLHelper::_('stylesheet', 'language/' . $lang->getTag() . '/' . $lang->getTag() . '.css', array('version' => 'auto'));
 
+// Adding ResponsifyJS
+HTMLHelper::_('script', 'media/vendor/responsifyjs/responsify.min.js', ['version' => 'auto', 'relative' => false]);
```

Comment:
this should not be changed, you need to load scripts for your plugin only, not the template